src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -54,24 +54,27 @@ internal class SolutionBindingOperation : ISolutionBindingOperation
         private readonly string projectName;
         private readonly SonarLintMode bindingMode;
         private readonly ILogger logger;
+        private readonly ILegacySonarQubeFolderModifier legacySonarQubeFolderModifier;
         private readonly IFileSystem fileSystem;
 
         public SolutionBindingOperation(IServiceProvider serviceProvider,
             ConnectionInformation connection,
             string projectKey,
             string projectName,
             SonarLintMode bindingMode,
-            ILogger logger)
-            : this(serviceProvider, connection, projectKey, projectName, bindingMode, logger, new FileSystem())
+            ILogger logger,
```

Comment:
The convention is to have the `logger` as the last parameter i.e. to have the "important"/task-specific parameters first.